locale/loris.pot

```diff
@@ -197,6 +197,9 @@ msgstr[0] ""
 msgid "Session"
 msgstr ""
 
+msgid "Instrument"
```

Comment:
loris.pot already has Instrument/Instruments. Is this branch up to date with the main branch?